src/System.Runtime.Numerics/tests/BigInteger/modpow.cs

```diff
@@ -170,30 +170,35 @@ public static void ModPow2Large1SmallInt_Threshold()
+        [InlineData("-283588760164723199492097446398514238808996680579814508529658395835708678571214111525627362048679162021949222615325057958783388520044013697149834530411072380435126870273057157745943859", "1042611904427950637176782337251399378305726352782300537151930702574076654415735617544217054055762002017753284951033975382044655538090809873113604", "11173562248848713955826639654969725554069867375462328112124015145073186375237811117727665778232780449525476829715663254692620996726130822561707626585790443774041565237684936409844925424596571418502946", "6662129352542591544500713232459850446949913817909326081510506555775206102887692404112984526760120407457772674917650956873499346517965094865621779695963015030158124625116211104048940313058280680420919")]
+        [InlineData("683399436848090272429540515929404372035986606104192913128573936597145312908480564700882440819526500604918037963508399983297602351856208190565527", "223751878996658298590720798129833935741775535718632942242965085592936259576946732440406302671204348239437556817289012497483482656", "1204888420642360606571457515385663952017382888522547766961071698778167608427220546474854934298311882921224875807375321847274969073309433075566441363244101914489524538123410250010519308563731930923389473186", "1136631484875680074951300738594593722168933395227758228596156355418704717505715681950525129323209331607163560404958604424924870345828742295978850144844693079191828839673460389985036424301333691983679427765")]
+        [InlineData("736513799451968530811005754031332418210960966881742655756522735504778110620671049112529346250333710388060811959329786494662578020803", "2461175085563866950903873687720858523536520498137697316698237108626602445202960480677695918813575265778826908481129155012799", "-4722693720735888562993277045098354134891725536023070176847814685098361292027040929352405620815883795027263132404351040", "4351573186631261607388198896754285562669240685903971199359912143458682154189588696264319780329366022294935204028039787")]
+        [InlineData("1596188639947986471148999794547338", "685242191738212089917782567856594513073397739443", "41848166029740752457613562518205823134173790454761036532025758411484449588176128053901271638836032557551179866133091058357374964041641117585422447497779410336188602585660372002644517668041207657383104649333118253", "39246949850380693159338034407642149926180988060650630387722725303281343126585456713282439764667310808891687831648451269002447916277601468727040185218264602698691553232132525542650964722093335105211816394635493987")]
+        [InlineData("-1506852741293695463963822334869441845197951776565891060639754936248401744065969556756496718308248025911048010080232290368562210204958094544173209793990218122", "64905085725614938357105826012272472070910693443851911667721848542473785070747281964799379996923261457185847459711", "2740467233603031668807697475486217767705051", "-1905434239471820365929630558127219204166613")]
```

Comment:
> Do they show up usefully in the test log?

I changed a single digit in one of them to see what it looks like when it fails:
```
System.Numerics.Tests.modpowTest.ModPow3LargeInt(value: "-1980488225727187161599876741331015456909269161086"..., exponent: "50051263260206609050451172764283809286167418846283"..., modulu
  s: "-1031394351020333660563633380634987137272001901984"..., expected: "-4559679593639479333979462122256288215243720737403") [FAIL]
        Assert.Equal() Failure
        Expected: -4559679593639479333979462122256288215243720737403
        Actual:   -78084019301444356054762453584152789691856362900873
        Stack Trace:
           c:\Users\stoub\Source\repos\corefx\src\System.Runtime.Numerics\tests\BigInteger\modpow.cs(195,0): at System.Numerics.Tests.modpowTest.ModPow3LargeInt(String value, String exponent,
   String modulus, String expected)
```
That seems reasonable to me both for compactness and ability to identify which one failed.

Similarly all of them show up reasonably in the results .xml file:
```
      <test name="System.Numerics.Tests.modpowTest.ModPow3LargeInt(value: &quot;16152447169612934532253858872860101823992426489763&quot;..., exponent: &quot;81750440863317178198977948957223170296434385667401&quot;..., modulus: &quot;-7196805437625531929619339290119232497987723600413&quot;..., expected: &quot;56163804698028541118832046136240625531208514124506&quot;...)" type="System.Numerics.Tests.modpowTest" method="ModPow3LargeInt" time="0.0198571" result="Pass">
        <traits>
          <trait name="category" value="outerloop" />
        </traits>
      </test>
```